README.md

```diff
@@ -1,2 +1,417 @@
-# Girls-First-Digital-Studio
-Girls First Digital Studio curriculum
+# Girls First Digital Studio
+## Variable - A storage of a value (number, words, etc)
+- Think of a variable like an envelope it holds anything you put inside of it.
+
+## Types of Variables - What specific type of values the variable can store.
+- Int - Stores integers (whole number negative and positive).
+    * public int year = 365;
+-  Boolean - Stores true or false values.
+    * public bool hired = true;
+-  Float - Stores decimal numbers.
+    * public float speed = .5f;
+
+
+
+
+
+
+
+
+# MICROSOFT WORKSHOP
+
+ 
+## Week 1: May 21 - May 22
+- Parents and Youth Orientation 
+-  Check-ins and Ice breakers 
+-  Introduction to Game Development and Unity 
+- Editor tour
+	* Basic navigation (e.g. Scene, Game, Directional Light, Console, Project, Asset Store, Inspector),
+	* Game objects/Components (e.g. RigidBody, Collider, Transform, Mesh Renderer)
+	* Building simple game from 3D primitives
+- Created objects Cube, players
+- Created a C# Script (CubeMove) for moving the cube in four directions
+-  Created Microsoft OneDrive Accounts to save projects
+
+
+
+
+
+
+
+
+
+
+
+
+# CubeMove Script:
+
+	using UnityEngine;
```

Comment:
Use syntax highlighting https://help.github.com/articles/creating-and-highlighting-code-blocks/